fili-core/src/main/java/com/yahoo/bard/webservice/logging/RequestLog.java

```diff
@@ -401,6 +401,18 @@ public static String getId() {
     }
 
     /**
+     * Set id prefix.
```

Comment:
Since it's no longer setting, but is instead appending (so subsequent calls will keep appending), this JavaDoc should make that more clear.